RadixWallet/Features/SettingsFeature/ManualAccountRecoveryScan/ManualAccountRecoverySeedPhrase+Reducer.swift

```diff
@@ -21,16 +21,16 @@ public struct ManualAccountRecoverySeedPhrase: Sendable, FeatureReducer {
 	public struct Destination: DestinationReducer {
 		@CasePathable
 		public enum State: Sendable, Hashable {
-			case enterSeedPhrase(ImportMnemonic.State)
```

Comment:
We call i `importMnemonic` in all other places, lets keep consistency :)

Reply:
Well, we don't generally have the policy of naming the case after the payload for Destination. My philosophy is that the case name should reflect the purpose, something that the payload type cannot always do, since it might be reused in different places.

On that topic, I suppose it shouldn't really be called ImportMnemonic at all, but let's change that later, to avoid merge conflicts.